xapian-bindings/python3/Makefile.am

```diff
@@ -30,51 +30,78 @@ RST_DOCS = \
 # Therefore, we don't run it by default currently.
 # replicationtest.py
 
-BUILT_SOURCES = xapian_wrap.cc xapian_wrap.h xapian.py
+BUILT_SOURCES = xapian_wrap.cc xapian_wrap.h xapian.py \
+	xapianletor_wrap.cc \
+	xapianletor_wrap.h \
+	xapianletor.py
 
-EXTRA_DIST = python.i util.i extra.i extracomments.i except.i \
+EXTRA_DIST = python.i pythonletor.i util.i utilletor.i extra.i extracomments.i except.i \
+	exceptletor.i \
 	doxy2swig.py \
 	fixup-swig-py3-wrapper \
 	generate-python-exceptions \
+	generate-letor-python-exceptions \
 	run-python-test \
 	sphinx-build \
 	testsuite.py \
 	test_xapian_star.py \
+	test_xapian_letor_star.py \
 	$(TESTS) $(BUILT_SOURCES) $(RST_DOCS)
 
 pkgpylibdir = @PYTHON3_LIB@/xapian
 
+letorpkgpylibdir = @PYTHON3_LIB@/xapianletor
```

Comment:
I'd probably keep all the `letorpkgpylib` stuff together, rather than interspersed with the core bindings as currently.

Reply:
fixed in a fixup commit.